src/user/controller/password.ts

```diff
@@ -0,0 +1,28 @@
+import Controller from '@curveball/controller';
+import { Context } from '@curveball/core';
+import * as UserService from '../service';
+import { Forbidden, Unauthorized, NotFound } from '@curveball/http-errors';
+import { User } from '../types';
+import * as privilegeService from '../../privilege/service';
+
+class UserPasswordController extends Controller {
+
+  async put(ctx: Context) {
+
+    if (!await privilegeService.hasPrivilege(ctx, 'admin')) {
+      throw new Forbidden('Only users with the "admin" privilege may create new users');
+    }
+
+    const userBody: any = ctx.request.body;
+    const user = await UserService.findByHref(ctx.params.href) as User;
```

Comment:
I don't think you have a `href` here. It's also better to not cast, because in this case it might be wrong. What if you got an app or a group?